al2/rust/hello/{{ cookiecutter.project_slug }}/README.md

```diff
@@ -33,8 +33,8 @@ To deploy the application, you need the folllowing tools:
 
 * SAM CLI - [Install the SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
 * Docker - [Install Docker community edition](https://hub.docker.com/search/?type=edition&offering=community)
-* [Rust](https://www.rust-lang.org/) version 1.56.0 or newer
```

Comment:
nit: Not part of the changes here, but noticed that there's a typo on line 27 `Requirments`.

Reply:
good catch